_posts/2023-03-01-release.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Release 2023.09
+author: Priya
+date: 2023-03-01
+category: Jekyll
+layout: post
+---
+
+### Enhancements
+
+#### Warning for Duplicate Foreign View Key Columns in Join Rule
+Mammoth now warns the user when applying a join rule, only if the key columns from the foreign view used have non-unique values. This warning will be shown even if the rows don't exceed the threshold. If the rows exceed the threshold and result in a join runtime error, this warning will also be displayed.
+
+#### Data Deletion upon Deletion of Contributing Actions
+When an action is deleted, the data contributed by it will also be deleted. For example, a batch contributed in the case of "Save as Ds -replace mode," the published database in the case of "publish to db."
+
+#### Back Button for Data Addition Modals
+Data addition modals now have a back button that redirects to the main menu.
+
+### Defect fixes
+
+* Addressed the issue where pipeline rules were being modified unexpectedly when changing the foreign view in join.
+* Resolved the issue where window function sort was becoming undefined upon duplicating the view.
+* Fixed the issue of column names not being generated correctly when changing data settings for CSV files in the file processor.
+* Resolved the issue where Apply template was not working on views.
+* Third-party modal: Fixed the issue of Text type columns being listed in the unique sequence column dropdown.
+* Third-party modal: The Schedule option no longer shows up from Google Drive and Google Dropbox.
```

Comment:
Rename Third-party to Connectors,
So,
Connectors Modal: 

Also
Connectors Modal: Schedule option was erroneously showing up for connectors where this is not applicable. This has been fixed.